addons/website_event/views/event_templates_list.xml

```diff
@@ -366,6 +366,9 @@
                             <t t-elif="event.event_registrations_sold_out">
                                 Sold Out
                             </t>
+                            <t t-elif="event.visibility == 'logged_users'">
+                                Logged Users Only
+                            </t>
```

Comment:
I don't see the need for this?

If you land here, you already found the event, and it will not prevent you from doing anything as it's just visibility based?